assets/setup-wizard/style.scss

```diff
@@ -1,26 +1,25 @@
+$small-screen-width: 600px;
+$small-vertical-padding: 20px;
+$small-horizontal-padding: 20px;
+$big-vertical-padding: 60px;
+$big-horizontal-padding: 80px;
+
 @import '../css/font-recoleta';
 @import 'fullscreen';
+@import 'animations';
 @import 'welcome/style';
+@import 'usage-tracking/style';
 @import 'progress-bar/style';
 
-$small-screen-width: 600px;
-
-@keyframes slideInFromBottom {
-	0% {
-		transform: translateY(20px);
-		opacity: 0;
-	}
-	100% {
-		transform: translateY(0);
-		opacity: 100%;
-	}
+.sensei-setup-wizard-page {
+	overflow-x: hidden;
+	background-color: #b7c9be;
```

Comment:
I moved this background to the `body` because when we have vertical scroll in the content, we would have some part white (without background) because the `height: 100vh;` in the `.sensei-setup-wizard`.